Week2/homework/map_filter.js

```diff
@@ -3,3 +3,5 @@
 const numbers = [1, 2, 3, 4];
 
 // Add your code here
+const newNum = numbers.filter(x => x % 2 !== 0).map(x => x * 2);
```

Comment:
Very nice.  Works perfectly. 
As you seem to enjoy reduce (as seen above) here is what I put together (using the `spread` operator, look it up 😛 )
```
const redDouble = numbers.reduce((acc, n) => (n % 2 !== 0 ? [...acc, n * 2] : acc), []);
```